message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -27,17 +27,7 @@
     </enum>
     <!-- ardupilot specific MAV_CMD_* commands -->
     <enum name="MAV_CMD">
-      <!-- 211 used by common.xml -->
-      <entry value="212" name="MAV_CMD_DO_AUTOTUNE_ENABLE" hasLocation="false" isDestination="false">
```

Comment:
I don't see PX4 Firmware using this command.

Is there a second user of this command which warrants it being in common?